src/main/java/org/kohsuke/github/GHRelease.java

```diff
@@ -122,18 +122,33 @@ GHRelease wrap(GHRepository owner) {
      * http://stackoverflow.com/questions/12361090/server-name-indication-sni-on-java but involve more complicated
      * handling of the HTTP requests to github's API.
      *
+     * @param file file to upload
+     * @param contentType content type
+     * @param uploadUrlPrefix ULR prefix for uploading; for example, for github.com it's https://uploads.github.com
      * @throws IOException
      */
```

Comment:
Makes sense to add <code>@since</code> in order to let others know which minimal API version provides this method